hack/bin/get-session-token

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+#set -x
+set -e -o pipefail
+
+WIRE_COOKIEJAR=/tmp/get-session-token.cookiejar
+
+#WIRE_HOST=https://prod-nginz-https.wire.com
+#WIRE_USER='...'
+#WIRE_PASSWORD='...'
```

Comment:
A bit more documentation would be nice.

Reply:
it's a hack.  it's meant to be used by editing the code, so you'll have to read and understand most of it anyway.  so adding docs would feel to me like adding noise.  what exactly do you have in mind?  maybe links to the swagger docs of the end-points called here?

Reply:
Thinking about this some more, how about an example flow we could copy paste to save ourselves even more time? 😄